source/connect-to-mongo/create-a-mongo-client.txt

```diff
@@ -0,0 +1,122 @@
+.. _java-rs-mongo-client:
+
+====================
+Create a MongoClient
+====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** instance, which creates the connection to the MongoDB deployment
+  and lets you perform operations on it.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoClient`` instance
+to connect to MongoDB.
+
+.. _java-rs-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies the URI as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+
+        .. TODO: uncomment out when java-rs-auth page made
+        .. For more information about the ``authSource`` connection option, see
+        .. :ref:`java-rs-auth`. 
+
+   * - ``host[:port]``
+
+     - Required. The host and port number where MongoDB is running. If you don't
+       include a port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. 
+        ..    TODO: uncomment out when connection options page made
+        ..    See :ref:`java-rs-connection-options` for a full description of
+        ..    these options.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string>` in the
+MongoDB Server documentation.
+
+MongoClient
+-----------
+
+To create a connection to MongoDB, pass a connection URI as a string to the
+``MongoClient`` constructor. In the 
```

Comment:
```suggestion
To create a connection to MongoDB, pass a connection URI as a string to the
method constructing a ``MongoClient``. In the 
```

The method `MongoClients.create` is not a "constructor". "constructor" has a specific different meaning in Java.

Reply:
Weirdly, I cannot resolve the conversations I started, so I will be marking them with

✅ Resolved.